src/create-client.ts

```diff
@@ -186,6 +186,7 @@ export async function createClient(
 
           if (authenticationResponse) {
             _authkitClientState = "AUTHENTICATED";
+            _scheduleAutomaticRefresh()
```

Comment:
this is due for a bit of a refactor (in the early days i would use `setInterval` to schedule the refresh when it was actually needed (that's not reliable though so i moved to the every 1s thing). should probably make this a setTimeout instead and let it run all the time.

(i want to get this minor tweak out for a bugfix release and then maybe i'll look at bigger changes).